gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/DagActionStoreChangeMonitor.java

```diff
@@ -154,9 +165,15 @@ protected void processMessage(DecodeableKafkaRecord message) {
         log.info("DagAction change ({}) received for flow: {}", dagActionType, dagAction);
         if (dagActionType.equals(DagActionStore.FlowActionType.RESUME)) {
           dagManager.handleResumeFlowRequest(flowGroup, flowName,Long.parseLong(flowExecutionId));
+          if (isMultiLeaderDagManagerEnabled) {
```

Comment:
should we remove this for now if we are also not doing launch? do u want to test only kill or kill and resume in first iteration?